### PR TITLE
CRM-17637 - Add poor man's cron fallback for VersionCheck

### PR DESCRIPTION
**Pull request by colemanw**
https://issues.civicrm.org/jira/browse/CRM-17637

### CODE REVIEW

File: CRM/Utils/VersionCheck.php [hunk 1]
```diff
@@ -82,20 +86,38 @@ class CRM_Utils_VersionCheck {
...
-    $localFile = Civi::paths()->getPath(self::LOCALFILE_NAME);
-    if (file_exists($localFile)) {
-      require_once $localFile;
-    }
-    if (function_exists('civicrmVersion')) {
-      $info = civicrmVersion();
-      $this->localVersion = trim($info['version']);
-      $this->localMajorVersion = $this->getMajorVersion($this->localVersion);
-    }
+    $this->localVersion = CRM_Utils_System::version();
```
Comment (totten) [inline CRM/Utils/VersionCheck.php]: Yay!

**Comment by totten** [comment]
Two observations. These are probably fine, but just to verbalize:

1. The poorman's cron here is job-specific (i.e. it only applies to version_check; not generally to `job.execute`). At a policy level, that might be good (since some cron jobs could be very slow)?
2. If cron is properly configured, but if the version_check job is disabled, then this would still execute the job, right?

**Comment by colemanw** [comment]
1. Yes, this is not a global poorman's cron.
2. No, it only pings back if the job is enabled. My thinking was that if a site admin has explicitly disabled the version_check job, we should respect their wishes.

**Comment by colemanw** [comment]
Unrelated fail.

**Pull request merged**